ScoutSuite/providers/base/provider.py

```diff
@@ -112,21 +110,18 @@ def _load_metadata(self):
         with open(self.metadata_path, 'rt') as f:
             self.metadata = json.load(f)
 
-    def _build_services_list(self, supported_services, services, skipped_services):
+    @staticmethod
+    def _build_services_list(supported_services, services, skipped_services):
         return [s for s in supported_services if (services == [] or s in services) and s not in skipped_services]
 
     def _update_last_run(self, current_time, ruleset):
-        last_run = {}
-        last_run['time'] = current_time.strftime("%Y-%m-%d %H:%M:%S%z")
-        last_run['cmd'] = ' '.join(sys.argv)
-        last_run['version'] = scout2_version
-        last_run['ruleset_name'] = ruleset.name
-        last_run['ruleset_about'] = ruleset.about
-        last_run['summary'] = {}
+        last_run = {'time': current_time.strftime("%Y-%m-%d %H:%M:%S%z"), 'cmd': ' '.join(sys.argv),
+                    'version': scout2_version, 'ruleset_name': ruleset.name, 'ruleset_about': ruleset.about,
```

Comment:
Could we put keys on different lines? I find it more readable when there are many keys or when the values are somewhat long.

Reply:
Hmm sorry, but I prefer it this way, find me someone else in the team who prefers the long way and I'll change it 😉